deap/tools/crossover.py

```diff
@@ -428,17 +443,62 @@ def cxESTwoPoint(ind1, ind2):
 
     return ind1, ind2
 
+
 def cxESTwoPoints(ind1, ind2):
     """
     .. deprecated:: 1.0
         The function has been renamed. Use :func:`cxESTwoPoint` instead.
     """
     return cxESTwoPoint(ind1, ind2)
 
+
+######################################
+# ES Crossovers                      #
+######################################
+
+def semantic_crossover(ind1, ind2, gen_func=gp.genGrow, pset=None, min=2, max=6):
```

Comment:
You should put this in the gp module, all operators are there (for the moment). It should also be named cxSemantic.